README.md

```diff
@@ -11,7 +13,8 @@ for the frontend extension.
 ## Requirements
 
 - Jupyter Server
-- \[optional\] JupyterLab >= 4.0.0
+- \[recommended\] Real-time collaboration for JupyterLab/Notebook:
+    This will push the kernels results in the notebook from the server.
```

Comment:
Linter has a problem here with too deep indent; https://github.com/datalayer/jupyter-server-nbmodel/pull/21 suggests to add an extra which does not include RTC but includes the packages needed for this extension to work.

```suggestion
   This will push the kernels results in the notebook from the server.
```